pandas/tests/io/test_excel.py

```diff
@@ -2377,12 +2377,9 @@ def check_called(func):
 
         with pd.option_context('io.excel.xlsx.writer', 'dummy'):
             register_writer(DummyClass)
-            writer = ExcelWriter('something.test')
+            writer = ExcelWriter('something.xlsx')
             assert isinstance(writer, DummyClass)
             df = tm.makeCustomDataframe(1, 1)
-
-            func = lambda: df.to_excel('something.test')
```

Comment:
Were these deleted for any particular reason? Think this should just be updated to the `.xlsx` extension here, no?

Reply:
See below at line 2386: `check_called(lambda: df.to_excel('something.xlsx'))`, updating the extension would make the line redundant.